draft-linus-trans-gossip-ct.md

```diff
@@ -216,18 +218,18 @@ before storing them:
   discarded
 
   1. if the leaf cert is not for a domain that the server is
-  authoritative for, the SCT MUST be discarded
+  authoritative for, the SCT MUST be discarded [why? perhaps the client doesn't care about privacy? e.g. it is a crawler]
```

Comment:
Well these rules are for the server. They're intended to mitigate the scalability requirements for a server, and only require them to store and serve a limited number of SCTs.

A client may indeed send all the SCTs it sees up somewhere because it doesn't care about privacy, but either that somewhere will either be an auditor (in which case I would call it a trusted auditor relationship) or it's a server that is ignoring these rules (in which case I would say that server is operating as a trusted auditor.)